apps/api/src/app/events/usecases/process-subscriber/process-subscriber.usecase.ts

```diff
@@ -90,10 +95,21 @@ export class ProcessSubscriber {
         firstName: subscriberPayload.firstName,
         lastName: subscriberPayload.lastName,
         phone: subscriberPayload.phone,
+        avatar: subscriberPayload.avatar,
```

Comment:
@scopsy I saw that the avatar was missing, was it intended?

Reply:
Nope :) good catch!